src/wms/src/main/java/org/geoserver/wms/capabilities/Capabilities_1_3_0_Transformer.java

```diff
@@ -1082,7 +1082,14 @@ private void handleStyles(final LayerInfo layer) {
                 }
                 element("Name", defaultStyle.prefixedName());
                 if (ftStyle.getDescription() != null) {
-                    element("Title", ftStyle.getDescription().getTitle());
+                    // PMT: WMS capabilities requires at least a title,
+                    // if description's title is null, use the name
+                    // for title.
+                    if (ftStyle.getDescription().getTitle() != null) {
+                        element("Title", ftStyle.getDescription().getTitle());
+                    } else {
+                        element("Title", defaultStyle.prefixedName());
+                    }
```

Comment:
If the description is null, the title will be missing. Probably want another else clause outside of `if (ftStyle.getDescription() != null)`